Prepare for Mergify built-in integration (ECOINT-20) (#2487)
DataDog/integrations-extras: mergify/README.md

## Setup

> You can either install the built-in integration, or install the agent.

Reviewer: ```suggestion
You can either install the built-in integration or install the agent.
```

> ### Setup Built-In Integration

Reviewer: ```suggestion
### Set up the built-in integration
```

> - **In Mergify dashboard**: Go to [Mergify dashboard][10], log in and navigate to **Integrations**.
> 
> - Click the **Connect** button in the **Datadog** tile.
> 
> ![Mergify Screenshot][12]
> 
> - You will be redirected to the Datadog authorization page.
> 
> - Log in to Datadog to complete setup.

Reviewer: ```suggestion
1. Go to the [Mergify dashboard][10], log in, and navigate to **Integrations**.
2. Click the **Connect** button in the **Datadog** tile and you will be redirected to the Datadog authorization page.
    ![Screenshot showing the Mergify dashboard][12]
3. Log in to Datadog to complete setup.
```

Your Mergify statistics now appear in Datadog.

> ### Setup Agent

Reviewer: ```suggestion
### Set up Agent
```

> #### From Release

Reviewer: ```suggestion
#### From release
```

> Run `datadog-agent integration install -t datadog-mergify==<INTEGRATION_VERSION>`.

Reviewer: ~~~suggestion
To set up the agent from release, run the following command, replacing `<INTEGRATION_VERSION>` with the version number for this integration:

```
datadog-agent integration install -t datadog-mergify==<INTEGRATION_VERSION>
```
~~~

> #### From Source

Reviewer: ```suggestion
#### From source
```

> To install the Mergify check on your host:

Reviewer: ```suggestion
To install the Mergify check on your host, run the following command, replacing `path/to/mergify/dist/` with the path to your Mergify distribution and `<ARTIFACT_NAME>` with the name of your artifact:
```

 run `datadog-agent integration install -w

>  path/to/mergify/dist/<ARTIFACT_NAME>.whl`.

Reviewer: ~~~suggestion
```
datadog-agent integration install -w path/to/mergify/dist/<ARTIFACT_NAME>.whl`
```
~~~

#### Configuration

1. Edit the `mergify.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][9] to start collecting your Mergify [metrics](#metrics).

   See the sample [mergify.d/conf.yaml.example][3] file for all available configuration options.

2. [Restart the Agent][4].

#### Validation

Run the [Agent's status subcommand][5] and look for `mergify` under the Checks section.

## Uninstallation

> - **In Mergify dashboard**: Go to [Mergify dashboard][10], log in and navigate to **Integrations**.
> 
> - Click the **Disconnect** button in the **Datadog** tile.
> 
> Once this integration has been uninstalled, any previous authorizations are revoked.
> 
> Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][11].

Reviewer: ```suggestion
1. Go to the [Mergify dashboard][10], log in, and navigate to **Integrations**.
2. Click the **Disconnect** button in the **Datadog** tile.

Once this integration has been uninstalled, any previous authorizations are revoked.

Additionally, ensure that all API keys associated with this integration are disabled by searching for the integration name on the [API Keys page][11].
```

[7]: https://github.com/DataDog/integrations-extras/blob/master/mergify/assets/service_checks.json
[8]: https://docs.datadoghq.com/developers/integrations/new_check_howto/#configure-the-developer-tool
[9]: https://docs.datadoghq.com/agent/guide/agent-configuration-files/#agent-configuration-directory
[10]: https://dashboard.mergify.com
[11]: https://app.datadoghq.com/organization-settings/api-keys?filter=Mergify

> [12]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/mergify/images/setup_integration.png

Reviewer: This link is 404ing. It looks like you haven't added the image to the folder.

Author: The image is in the pull request: https://github.com/DataDog/integrations-extras/pull/2487/files#diff-efb725effa0f253201963423f0acf815ebed45752da1028745bde91688aa1a60
